services/bots/github_apps.py

```diff
@@ -169,6 +169,26 @@ def get_specific_github_app_details(
     )
 
 
+def _filter_rate_limited_apps(
+    apps_to_consider: List[GithubAppInstallation],
+) -> List[GithubAppInstallation]:
+    redis_connection = get_redis_connection()
+    return list(
+        filter(
+            lambda obj: not is_installation_rate_limited(
+                redis_connection, obj.installation_id, app_id=obj.app_id
+            ),
+            apps_to_consider,
+        )
+    )
+
+
+def _filter_suspended_apps(
```

Comment:
Optional: making tests for this that isn't necessarily tethered to the notify test